raft/testdata/campaign.txt

```diff
@@ -29,15 +29,15 @@ stabilize
   Lead:0 State:StateCandidate
   HardState Term:1 Vote:1 Commit:2
   Messages:
-  1->2 MsgVote Term:1 Log:1/2
-  1->3 MsgVote Term:1 Log:1/2
+  1->2 MsgVote Term:1 Log:1/2 Commit:2
```

Comment:
We should probably extend the `String()` method to include the commit term and not just the commit.